arch/xtensa/src/esp32/esp32_spiflash.c

```diff
@@ -1570,9 +1570,7 @@ static ssize_t esp32_read(struct mtd_dev_s *dev, off_t offset,
   finfo("esp32_read()=%d\n", ret);
 #endif
 
-error_with_buffer:
-
-  return (ssize_t)ret;
+  return ret;
```

Comment:
Notice: Maybe ret should be changed to ssize_t?

Reply:
Done.